src/claims/claims.controller.ts

```diff
@@ -77,4 +80,64 @@ export class ClaimsController {
 
     return handleServiceResult(serviceResult);
   }
+
+  @ApiOperation({
+    summary: 'Register a CustomClaimType',
+    description: 'This endpoint will add the CustomClaimType to the network',
+  })
+  @ApiTransactionResponse({
+    description: 'Transaction response',
+    type: TransactionQueueModel,
+  })
+  @ApiTransactionFailedResponse({
+    [HttpStatus.BAD_REQUEST]: [
+      'Validation: CustomClaimType name length exceeded',
+      'Validation: The CustomClaimType with provided name already exists',
+    ],
+  })
+  @Post('custom-claim-type')
+  async registerCustomClaimType(
+    @Body() args: RegisterCustomClaimTypeDto
+  ): Promise<TransactionResponseModel> {
+    const serviceResult = await this.claimsService.registerCustomClaimType(args);
+
+    return handleServiceResult(serviceResult);
+  }
+
+  @ApiOperation({
+    summary: 'Get CustomClaimType',
+    description:
+      'This endpoint will retrieve the custom claim type by id or name. <br /> `ID` or name needs to be provided. <br/> If both are provided will fetch using `ID`',
```

Comment:
```suggestion
      'This endpoint will retrieve the custom claim type by id or name. <br /> ID or name needs to be provided. <br/> If both are provided will fetch using ID',
```

(nit) when I see the backticks I read it as a literal thing I should copy paste. 

If using them I'd write `id` or `name`, but I don't think they are needed.